annotation/src/main/kotlin/com/tobrun/datacompat/annotation/DataCompat.kt

```diff
@@ -3,7 +3,10 @@ package com.tobrun.datacompat.annotation
 /**
  * Annotation class of DataCompat.
  * Classes annotated with this annotation are required to be Kotlin data classes with private visibility.
+ *
+ * @param imports if any constructor values require additional imports, they should be passed here.
+ *  E.g. `["android.graphics.Color"]`
  */
 @Retention(AnnotationRetention.RUNTIME)
 @Target(AnnotationTarget.CLASS)
-annotation class DataCompat
+annotation class DataCompat(val imports: Array<String> = [])
```

Comment:
can we rename it to `importsForDefaults`. It might be misleading and folks might think these imports are needed for the property type itself instead of the code in the default

Reply:
👍